### PR TITLE
PR : Result Page

### PR DESCRIPTION
**Pull request by pavan6115**
- Built Result Page

Hnet com-image

### CODE REVIEW

File: pages/jdm/result.css [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+
+.space {
+    height: 2rem;
+    width: 1rem;
+    margin-top: 3rem;
+}
+
+.answer__btn:hover {
+    background-color: var(--primary-color);
+    color: white;
```
Comment (saketkum11) [inline pages/jdm/result.css:219]: I am not sure maybe you have missed the color variable.

**Review by saketkum11 — APPROVED** [review]
Good to  go ahead.

**Review by Nishant-96 — APPROVED** [review]
Looks good to merge. just remove the commented unwanted code pieces.

File: pages/jdm/result.css (hunk 1 above)
Comment (pavan6115) [reply pages/jdm/result.css:219]: ok

**Pull request merged**